notification-portlet-webcomponents/notification-modal/src/main.js

```diff
@@ -1,4 +1,5 @@
-import "@babel/polyfill";
+import 'promise-polyfill/src/polyfill';
```

Comment:
Web Components present many of the same challanges as libraries, when loaded on a page.

---

> This is useful it you are building a website and want to support older browsers. Javascript library authors should NOT use this method.

```
import 'promise-polyfill/src/polyfill';
```

> If you would like to not affect the global environment (sometimes known as a ponyfill), you can import the base module. This is nice for library authors or people working in environment where you don't want to affect the global environment.

```
import Promise from 'promise-polyfill';
```

Source: https://github.com/taylorhakes/promise-polyfill

---

The Ponyfill looks like it should scope things and avoid polyfill conflicts.

Reply:
This causes it to break, regardless of where I put it.

Reply:
🤔 what about leveraging http://bluebirdjs.com/docs/getting-started.html as the promise ponyfill?